tasks/src/reinforcement-learning/data.ts

```diff
@@ -0,0 +1,34 @@
+import type { TaskDataCustom } from "../Types";
+
+const taskData: TaskDataCustom = {
+	datasets: [
+		{
+			description: "A curation of widely used datasets for Data Driven Deep Reinforcement Learning (D4RL)",
+			id:          "edbeeching/decision_transformer_gym_replay",
+		}
+	],
+    demo:     {
+		inputs:  [],
+		outputs: [],
+	},
+    metrics: [{
+        description: "Average return obtained after running the policy for a certain number of evaluation episodes ",
+        id:          "Mean Reward",
+    }],
+	models: [
+        {
+			description: "A Transformer-based Reinforcement Learning model trained on expert data from the Gym Hopper environment",

```

Comment:
We tried to avoid using terms that much to lower barrier of entry 🙂
```suggestion
			description: "A Reinforcement Learning model trained on expert data from the Gym Hopper environment",

```